pandas/tests/arithmetic/conftest.py

```diff
@@ -240,5 +240,14 @@ def box_with_array(request):
     return request.param
 
 
+@pytest.fixture(params=[pd.Index, pd.Series, tm.to_array, np.array, list], ids=id_func)
```

Comment:
i would name this box_1d_array as this is the purpose

Reply:
Done